backend/internetarchive/internetarchive.go

```diff
@@ -797,10 +803,23 @@ func (o *Object) Update(ctx context.Context, in io.Reader, src fs.ObjectInfo, op
 		"x-archive-queue-derive":     "0",    // skip derivation process (e.g. encoding to smaller files, OCR on PDFs)
 		"x-archive-cascade-delete":   "1",    // enable "cascate delete" (delete all derived files in addition to the file itself)
 	}
+
 	if size >= 0 {
 		headers["Content-Length"] = fmt.Sprintf("%d", size)
 		headers["x-archive-size-hint"] = fmt.Sprintf("%d", size)
 	}
+
+	for _, v := range o.fs.opt.Metadata {
+		// Split the metadata key and value
```

Comment:
If you could just note that this is the item metadata in this comment then that will save confusion later!